.github/workflows/unit_tests.yml

```diff
@@ -0,0 +1,78 @@
+name: unit tests
+
+on: [push, pull_request]
```

Comment:
Can we add a cron trigger here too? Similar to whatever weekly build we have for travis-ci. And we should turn off the travis-ci integration when we switch over.

Reply:
Please create a follow up issue to set up slack notifications for builds (if you haven't already)

Reply:
I'll see if I can make it part of this issue - if it's not trivial, I'll split it out.